docs/docker/import_data.md

```diff
@@ -0,0 +1,80 @@
+## Import data instructions ##
+This is an example to import a sample study: `study_es_0`.
+When trying to import other studies, please follow the same routine:
+1. import gene panels (if applicable, studies without gene panels are assumed to be whole exome/genome)
+2. import study data
+
+### Step 1 - Import gene panels
+
+To import gene panels for your study, please reference the example commands in [this file](example_commands.md#importing-gene-panel)
+
+These are the commands for importing `study_es_0` gene panels (`data_gene_panel_testpanel1` and `data_gene_panel_testpanel2`):
+```shell
+docker run -it --rm \
+    --net cbio-net \
+    -v /<path_to_config_file>/portal.properties:/cbioportal/portal.properties:ro \
+    cbioportal/cbioportal:latest \
+    bash -c 'cd /cbioportal/core/src/main/scripts/ && ./importGenePanel.pl --data /cbioportal/core/src/test/scripts/test_data/study_es_0/data_gene_panel_testpanel1.txt'
+```
+
+```shell
+docker run -it --rm \
+    --net cbio-net \
+    -v /<path_to_config_file>/portal.properties:/cbioportal/portal.properties:ro \
+    cbioportal/cbioportal:latest \
+    bash -c 'cd /cbioportal/core/src/main/scripts/ && ./importGenePanel.pl --data /cbioportal/core/src/test/scripts/test_data/study_es_0/data_gene_panel_testpanel2.txt'
+```
+
+### Step 2 - Import data
+
+To import data for your study, please reference the example commands in [this file](example_commands.md#importing-data)
+
+Command for importing `study_es_0` data:
+
+```shell
+docker run -it --rm --net cbio-net \
+    -v /<path_to_config_file>/portal.properties:/cbioportal/portal.properties:ro \
+    cbioportal/cbioportal:latest \
+    metaImport.py -u http://cbioportal-container:8080 -s /cbioportal/core/src/test/scripts/test_data/study_es_0
+```
+
+:warning: after importing a study, remember to restart `cbioportal-container`
+to see the study on the home page. Run `docker restart cbioportal-container`.
+
```

Comment:
Maybe good to have some concluding notes at the end. E.g.:

You have now imported the test study `study_es_0`. This study is included inside the cbioportal container. The process for adding a study that is outside of the container is almost the same. The difference is that one needs to add the `-v` parameter to mount the folder with the study data inside the container. See `How to import a real study` section (add markdown link) for an example.